### PR TITLE
How to deploy the cluster autoscaler

### PR DESCRIPTION
**Pull request by ingvagabund**
Really basic info describing how to deploy the autoscaler and how to exercise the autoscaling capability.

### CODE REVIEW

**Comment by ingvagabund** [comment]
@frobware @vrutkovs @sdodson PTAL

File: playbooks/openshift-cluster-autoscaler/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+The autoscaler can be deployed directly by running the corresponding playbook:
+
+```sh
+ansible-playbook -i inventory playbooks/openshift-cluster-autoscaler/config.yml
+```
+
+or during your cluster deployment by setting `openshift_cluster_autoscaler_install=True`
+in the corresponding inventory file.
+
+Currently, the autoscaler is deployed only on nodes that have the `infra` [node role](https://github.com/openshift/openshift-ansible#node-group-definition-and-mapping) set.
```
Comment (vrutkovs) [inline playbooks/openshift-cluster-autoscaler/README.md]: Mention `openshift_cluster_autoscaler_node_selector`?

**Comment by ingvagabund** [comment]
@vrutkovs @frobware PTAL

**Comment by ingvagabund** [comment]
Just docs -> merging

**Pull request merged**